cybershell/payload_manager.py

```
@@ -0,0 +1,614 @@
+                )
+                if not tech_match:
+                    return 0.0  # Complete mismatch
+        
+        # Product matches, check version
+        if payload.version_spec and fingerprint.version:
+            try:
+                if payload.matches_version(fingerprint.version):
+                    return 1.0  # Perfect version match
+                else:
+                    return 0.2  # Product matches but version doesn't
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)